src/courtroom.cpp

```diff
@@ -2323,7 +2323,25 @@ void Courtroom::display_pair_character(QString other_charid, QString other_offse
       // Move pair character according to the offsets
       ui_vp_sideplayer_char->move(ui_viewport->width() * offset_x / 100,
                                   ui_viewport->height() * offset_y / 100);
-
+      // Split the charid according to the ^ to determine if we have "ordering" info
+      QStringList args = other_charid.split("^");
+      if (args.size() >
+          1) // This ugly workaround is so we don't make an extra packet just
+              // for this purpose. Rewrite pairing when?
+      {
```

Comment:
When there is no ordering info, is the stacking simply undefined? What is supposed to happen?

Reply:
The player that is talking will be layered over the player that is not talking.